WalletWasabi.Fluent/Views/Wallets/MusicControlsView.axaml

```diff
@@ -5,11 +5,36 @@
              xmlns:wallets="clr-namespace:WalletWasabi.Fluent.ViewModels.Wallets"
              xmlns:i="using:Avalonia.Xaml.Interactivity"
              xmlns:behaviors="clr-namespace:WalletWasabi.Fluent.Behaviors"
+             xmlns:converters="clr-namespace:WalletWasabi.Fluent.Converters"
              Background="Transparent"
              x:DataType="wallets:WalletViewModel"
              x:CompileBindings="True"
              mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="64"
              x:Class="WalletWasabi.Fluent.Views.Wallets.MusicControlsView">
+  <UserControl.Resources>
+    <converters:KeyToDictionaryItemConverter x:Key="MusicBoxMessageConverter">
+      <ResourceDictionary>
+        <StackPanel x:Key="{x:Static wallets:CoinJoinStateViewModel.PressPlayToStartMessage}">
+          <TextBlock Inlines="">
+            <Run>Press </Run>
+            <InlineUIContainer>
+              <PathIcon Data="{StaticResource play_regular}" />
+            </InlineUIContainer>
+            <Run> to start</Run>
+          </TextBlock>
+        </StackPanel>
+        <StackPanel x:Key="{x:Static wallets:CoinJoinStateViewModel.PlebStopMessageBelow}">
+          <TextBlock Inlines="">
+            <Run>Add more funds or press </Run>
+            <InlineUIContainer>
+              <PathIcon Data="{StaticResource play_regular}" />
+            </InlineUIContainer>
+            <Run> to bypass</Run>
+          </TextBlock>
+        </StackPanel>
+      </ResourceDictionary>
+    </converters:KeyToDictionaryItemConverter>
+  </UserControl.Resources>
```

Comment:
This part bothers me, as the message is duplicated or at least the one in the ViewModel is useless...

What do you think about having a converter that generates Content from String?

For example, the string `Press Play to start` string is bound to the ContentControl in XAML, with a converter.
The convert in this case would start to build the content programmatically.

Basically split the string by space, so we have a list of words, then just iterate through them.

```c#
// Create a TextBlock
// Init the inlines

if (word == "Play")
   // add the Play icon to TextBlock.Inlines
else
  // add a TextBlock with the word to TextBlock.Inlines
  
Return TextBlock;
```

With we would be more flexible, the messages won't be duplicated plus we can change the messages more easily and dynamically.
Also, we could do the same for Stop or Pause icon.

WDYT?

Reply:
Nice! I like your approach. I've already implemented it. Please tell me if it's as you expected.